Documentation/docpage/input.html

```diff
@@ -259,85 +259,111 @@ <h2 id="input-validation" class="mg-2-top mg-1-bot title-lg-wt-5">
+
+                <!-- outline wrong password validation -->
+                <div class="outline-password-input wrong-password">
+                  <label for="password"
+                    >Password *
+                    <input
+                      type="password"
+                      name="password"
+                      autocomplete="current-password"
+                      required
+                      value="..."
+                    />
+                    wrong password, try again
                   </label>
                 </div>
-                <div class="signin-btn">
-                  <button type="submit" class="btn primary-btn-md">
-                    Submit
-                  </button>
+              </div>
+              <script type="text/plain" class="language-html snippet-container">
+                <!-- outline wrong input email validation -->
+                <div class="outline-email-input wrong-input">
+                  <label for="email">Email *
+                    <input type="email" name="email" value="abc.com" required autocomplete="email" />
+                    entered email is invalid
+                  </label>
                 </div>
-              </form>
-            </div>
-          </div>
 
-          <script type="text/plain" class="language-html snippet-container">
-            <!-- input validation -->
-            <div class="signin-modal modal-for-docs">
-              <form>
-                <div class="outline-email-input">
-                  <label for="email">
-                    <div class="input-icon">
-                      <i class="far fa-envelope"></i>
-                      <input
-                        type="email"
-                        name="email"
-                        required
-                        autocomplete="email"
-                        placeholder="Enter your email *"
-                      />
-                    </div>
+                <!-- outline wrong password validation -->
+                <div class="outline-password-input wrong-password">
+                  <label for="password">Password *
+                    <input type="password" name="password" autocomplete="current-password" required value="..." />
+                    wrong password, try again
                   </label>
                 </div>
-                <div class="outline-password-input">
-                  <label for="password">
-                    <div class="input-icon">
-                      <i class="fas fa-key"></i>
-                      <input
-                        type="password"
-                        name="password"
-                        autocomplete="current-password"
-                        placeholder="Enter your password *"
-                        required
-                      />
-                    </div>
+              </script>
+
+              <!-- success validation -->
+              <h2 class="title-md-wt-5 pd-2-top pd-1-bot">
```

Comment:
suggestion: make naming of classes self-explanatory it will help user use it easily